### ISSUE TITLE
Emoji auto-completion in Emoji layer core dumps Emacs

### ISSUE THREAD

**Issue by romatthe** [syl20bnr/spacemacs#10706]
#### Description :octocat:
Company-emoji seems to crash Emacs

#### Reproduction guide :beetle:
- Start Emacs
- Enable the `emoji` layer in spacemacs configuration
- Open any buffer with `emoji-cheat-sheet-plus-display-mode` enabled
- Type in anything that causes `company-emoji` to prompt, usually `:` followed by a letter like `t`
- My first encounter with this bug was when I was adding this to the top of an `org-mode` buffer: `#+OPTIONS: toc:nil \n:t`. Typing out the `t` causes Emacs to crash every single time.

*Observed behaviour:* :eyes: :broken_heart:
Emacs process core dumps instantly.

*Expected behaviour:* :heart: :smile:
Expected the `company-emoji` list to appear without issues.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 25.3.1
- Spacemacs: 0.200.13
- Spacemacs branch: master (rev. cce2a4b8)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(yaml auto-completion emacs-lisp emoji
      (erc :variables erc-server-list
           '(("irc.freenode.net" :port "6697" :ssl t :nick "romatthe")))
      git
      (haskell :variables haskell-completion-backend 'intero haskell-enable-hindent-style "johan-tibell")
      helm html latex
      (markdown :variables markdown-command "pandoc")
      (org :variables org-enable-github-support t)
      (shell :variables shell-default-height 30 shell-default-position 'bottom shell-default-term-shell "/bin/zsh")
      syntax-checking version-control)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND DBUS GCONF GSETTINGS NOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 MODULES XWIDGETS

#### Backtrace :paw_prints:
Unfortunately, `toggle-debug-on-error` (`SPC t D`) didn't allow me to get a backtrace, but I can provide a core dump log from `journalctl` if helpful.

**Comment by antonalekseev** [comment]
Isn't Emacs/Spacemacs issue, see #10695.

**Comment by romatthe** [comment]
Somehow managed to miss that issue, sorry about that. I searched through the issue tracker, didn't find anything, but only submitted this issue when I came back home later.

I'll be closing this one then. I simply turned off Emoji-layer. I don't care that much about it, but I thought it was worth reporting considering this was something I could consistently replicate. No need to pollute the issue tracker any further.
